pytorch_lightning/logging/wandb.py

```diff
@@ -0,0 +1,116 @@
+"""
+Log using `W&B <https://www.wandb.com>`_
+
+.. code-block:: python
+
+    from pytorch_lightning.logging import WandbLogger
+    wandb_logger = WandbLogger(
+        name="my_new_nun",    # Optional, display name
```

Comment:
If all are optimal, there is no need to write it

Reply:
Here the intent was just to show sample use if anybody wanted to use those args as it does not appear in the doc and they would have to look at the docstring of `logging.wandb.WandbLogger`